backend/tests/core/test_auth_utils.py

```
@@ -0,0 +1,317 @@
+class TestIsApiKey:
+    """Test is_api_key detection function"""
+
+    def test_api_key_with_wg_prefix(self):
+        """Test that tokens starting with 'wg-' are detected as API keys"""
+        assert is_api_key("wg-abc123") is True
+        assert is_api_key("wg-test-api-key-12345") is True
+        assert is_api_key("wg-") is True
+
+    def test_jwt_token_not_detected_as_api_key(self):
+        """Test that JWT tokens are not detected as API keys"""
+        jwt_token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJ0ZXN0In0.test"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "jwt_token"

(S105)